src/test/java/racing/RacingSessionTest.java

```diff
@@ -0,0 +1,61 @@
+package racing;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class RacingSessionTest {
+
+    @Test
+    @DisplayName("입력받은 수만큼 턴 행동")
+    void 턴_행동() {
+        RacingSession session = RacingSession.of("a,b");
+
+        List<CarPositions> positions = session.startRacing(5);
+
+        assertThat(positions).hasSize(5);
+        for (CarPositions position : positions) {
+            assertThat(position.getCarPositions()).hasSize(2);
+        }
+    }
+
+    @Test
+    @DisplayName("각 자동차 이동")
+    void 자동차_이동() {
+        List<Car> cars = List.of(Car.of("a"), Car.of("b"), Car.of("c"));
+
+        CarPositions positions = RacingSession.moveCars(cars);
+
+        assertThat(positions.getCarPositions()).hasSize(3);
+    }
+
+    @ParameterizedTest
+    @CsvSource(value = {"0:false", "1:false", "2:false", "3:false", "4:true", "5:true",
+            "6:true", "7:true", "8:true", "9:true"}, delimiter = ':')
+    void 전진_조건(int number, boolean canMove) {
+        assertThat(RacingSession.canProgress(number)).isEqualTo(canMove);
+    }
+
+    @Test
+    @DisplayName("경기 기록에서 우승자 찾기")
+    void 우승자_찾기() {
+        List<String> carNames = List.of("a","b", "c");
+
+        List<CarPositions> playHistory = List.of(
+            new CarPositions(List.of(1,1,1)),
```

Comment:
```suggestion
        List<String> carNames = List.of("a","b", "c");

        List<CarPositions> playHistory = List.of(
            new CarPositions(List.of(1,1,1)),
```
`1,1,1` 이 "a","b","c" 를 매칭시키기 위한 index를 매번 가지고 다녀야하는 이슈가 있을것같네요!
나중에 로직이 더 복잡해지면 이러한 개념은 사람이 이해하기 힘든 부분이 될수도 있다고 생각하는데요,

이러한 로직을 어떤객체에게 부여하고 객체간의 협력관계를 어떻게 형성할지에대해 고민해보셔도 좋을것같네요!

Reply:
@wooobo 
index 를 매번 가지고 다녀야 한다는 부분이 어떤 의미인지 잘 모르겠습니다.
List.of(1,1,1) 에서 a,b,c 의 순서를 외우고 있어야 한다는 의미일까요?

일단 가독성을 좀 올리기 위해서 변수명 등을 수정했습니다.